### ISSUE TITLE
Integrating with Third Party Extensions

### ISSUE THREAD

**Issue by cmiles74** [gildas-lormeau/SingleFile#294]
I'm working on an extension and wanted to use SingleFile to download the contents of the page loaded into a particular tab in Firefox. I followed the instructions in the wiki...

[How to Integrate the API of SingleFile into an Extension](
https://github.com/gildas-lormeau/SingleFile/wiki/How-to-integrate-the-API-of-SingleFile-into-an-extension)

This isn't working for me and I was wondering if these instructions are up-to-date. I'm seeing errors like the following in my content script...

> TypeError: "this.singlefile.extension.lib.fetch.content.resources is undefined"

And I'm not sure what I should be using for the "options" parameter on the content script side.

Any help would be greatly appreciated. Thank you!

**Comment by gildas-lormeau** [comment]
I tested the instructions and it works fine for me. Here is a zip file containing a simple example of extension using SingleFile. It displays in the console of any tab the content of the page (processed by SingleFile) after it has been loaded.

[singlefile - lite.zip](https://github.com/gildas-lormeau/SingleFile/files/3764502/singlefile.-.lite.zip)

**Comment by gildas-lormeau** [comment]
@cmiles74 I'm closing this issue because I was not able to reproduce it and I did not get any feedback from you. Feel free to re-open it if necessary.

**Comment by cmiles74** [comment]
Fair enough. I didn't have a lot of time yesterday but I will get to testing it soon, having the example code will surely make a huge difference. Thank you for your help with this! `:-)`

**Comment by gildas-lormeau** [comment]
No problem. Don't hesitate to post a message here if you need help or have any questions.
